### DISCUSSION TITLE
¿Qué diseño implementamos?

### DISCUSSION THREAD

**Discussion by bernat13** [VerneDevelopers/TriatGames#74] [Ideas]
En la creación de las vistas, se han creado 3 diseños distintos. ¿Cual os gusta más?

Comenta en los comentarios las ideas que te gusten de uno y de otro y si mezclarias ideas de varios.

![Captura de pantalla de 2024-02-09 20-59-48](https://github.com/VerneDevelopers/TriatGames/assets/4863914/f293ad97-50bb-45d1-a60e-3ece6e14b360)
![Captura de pantalla de 2024-02-09 20-59-37](https://github.com/VerneDevelopers/TriatGames/assets/4863914/e85d78b8-1aef-4447-bf01-496ea7701cbe)
![Captura de pantalla de 2024-02-09 20-59-22](https://github.com/VerneDevelopers/TriatGames/assets/4863914/28057be9-4a89-4e38-9839-981190dd5b8e)
![Captura de pantalla de 2024-02-09 20-59-15](https://github.com/VerneDevelopers/TriatGames/assets/4863914/8533d994-07db-4448-a654-8568db97a910)
![Captura de pantalla de 2024-02-09 20-58-59](https://github.com/VerneDevelopers/TriatGames/assets/4863914/a0788254-e381-457e-9dd7-737fc1e47beb)
![Captura de pantalla de 2024-02-09 20-58-49](https://github.com/VerneDevelopers/TriatGames/assets/4863914/2668459b-19d5-47cb-b578-2b968b8140b0)
![Captura de pantalla de 2024-02-09 20-57-19](https://github.com/VerneDevelopers/TriatGames/assets/4863914/833839d5-25ea-4619-9793-92a29291def4)
![Captura de pantalla de 2024-02-09 21-02-21](https://github.com/VerneDevelopers/TriatGames/assets/4863914/c94aac23-9150-4c4d-8936-e15c8c3cbd2d)
![Captura de pantalla de 2024-02-09 21-02-11](https://github.com/VerneDevelopers/TriatGames/assets/4863914/ba2eeab6-2850-4a70-b93c-0d67445f691f)

**Comment by CasadoManuelJesus** [comment]
La vista del wordle está increíble, la verdad.
